### DISCUSSION TITLE
Update field on a bunch of internal db records

### DISCUSSION THREAD

**Discussion by KristofferBroberg** [Budibase/budibase#9972] [Help] [answered]
I´ve imported approx. 400 records via CSV to the internal db. Now I wish to update a relationship field for each of these records.  I could of course do this manually, but I´d rather not. So, as someone proposed somewhere, set up an automation to do the job! Spent a couple of hours trying to no avail...

Is this even a feasible solution? If so, how should one set up this automation?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @KristofferBroberg,

Sorry for the delayed response on this, I meant to reply to this yesterday.

I have created the automation you require, screenshots of this below.
This being said currently there is an issue with the automations. Which will be fixed when the following is merged in: https://github.com/Budibase/budibase/pull/9984

**Create an app action automation**
<img width="514" alt="Screenshot 2023-03-14 at 10 53 41" src="https://user-images.githubusercontent.com/126772285/224980038-787f60f6-c795-421e-b717-ffcd1ce16248.png">

**Select the table you would like to query**
<img width="512" alt="Screenshot 2023-03-14 at 10 53 47" src="https://user-images.githubusercontent.com/126772285/224980274-5ac299e8-7d10-4453-8763-2ced6eaaa32a.png">

**Add looping**
<img width="530" alt="Screenshot 2023-03-14 at 10 53 58" src="https://user-images.githubusercontent.com/126772285/224981544-df806ccc-9e43-4021-9490-be85c7444b3e.png">

**Select the table**
<img width="492" alt="Screenshot 2023-03-14 at 10 54 11" src="https://user-images.githubusercontent.com/126772285/224981627-102fccb4-8a13-42b4-93c6-53b161c329e5.png">

**Make sure to add the binding _id of the relationship you want to add and pasted it into the correct field.**
<img width="484" alt="Screenshot 2023-03-14 at 10 54 31" src="https://user-images.githubusercontent.com/126772285/224981897-4e896987-1412-47c1-b960-2a72f5dbcf90.png">

Then run and it should add the relationship to all your records. 

I hope this helps!
